test/functional/ex_cmds/source_spec.lua

```diff
@@ -0,0 +1,51 @@
+local helpers = require('test.functional.helpers')(after_each)
+local command = helpers.command
+local insert = helpers.insert
+local wait = helpers.wait
+local eq = helpers.eq
+local clear = helpers.clear
+local meths = helpers.meths
+local feed = helpers.feed
+local feed_command = helpers.feed_command
+
+describe(':source', function()
+  before_each(function()
+    clear()
+  end)
+
+  it('current buffer', function()
+    insert('let a = 2')
+    wait()
```

Comment:
I don't think `wait()` should be used here. ( `wait()`  is the most misnamed function in the codebase, it rather means `cause_extra_rpc_activity()`.  it does not wait for anything that that `command()` or any sync RPC already waits for)

Reply:
I have removed the `wait()` calls.